.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -1,19 +1,19 @@
 ---
-name: "\U0001F41BBug report\U0001F41B"
-about: Create a report only if the developers asked you to do so
+name: "Please do not open an issue unless asked to do so by a developer in a discussion (see below)"
+about: Please include a link to the discussion in the first post.
 label: bug
```

Comment:
Please remove this label because we do not know if it is a bug report or a feature request if we just have one single template.

Reply:
I don't know if it can be removed.  But, I changed it to issue.

Reply:
Yes it can be removed or set to one or more labels we have defined at https://github.com/NightscoutFoundation/xDrip/labels. Please change it to 'status-triage', a newly created label.

Reply:
Done